### PR TITLE
[IMP] web_widget_x2many_graph: This module now can be used on 10.0 task#9512

### PR DESCRIPTION
**Pull request by keylor2906**
Using JSON instead of One2ManyField and it works smoothly without problem and more dynamic
Using less instead of css to look better

Examples:
image
image
image

### CODE REVIEW

File: web_widget_x2many_graph/static/src/js/web_widget_x2many_graph.js [hunk 1]
```diff
@@ -18,86 +18,43 @@
...
-            this.attrs = node.attrs;
-        },
-        start: function() {
-            this._super.apply(this, arguments);
-        },
-        get_value: function () {
-            var value = this.get('value');
-            return value
-        },
+    var FieldX2ManyGraphWidget = form_common.AbstractField.extend(form_common.ReinitializeFieldMixin, {
```
Comment (nhomar) [inline web_widget_x2many_graph/static/src/js/web_widget_x2many_graph.js]: Call it json-graph or something like that due to the refactor this is not anymore a X2MAnyGraph

File: web_widget_x2many_graph/static/src/js/web_widget_x2many_graph.js [hunk 2]
```diff
@@ -18,86 +18,43 @@
...
             });
         },
         destroy: function () {
             return this._super();
         },
     });
-}
+    core.form_widget_registry.add('x2many_graph', FieldX2ManyGraphWidget);
+
+});
```
Comment (nhomar) [inline web_widget_x2many_graph/static/src/js/web_widget_x2many_graph.js]: EOL fix please.

File: web_widget_x2many_graph/static/src/less/web_widget_x2many_graph.less [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.oe_form_field_x2many_graph {
```
Comment (nhomar) [inline web_widget_x2many_graph/static/src/less/web_widget_x2many_graph.less]: rename and re-document the way it works.

no more references to x2many

**Comment by nhomar** [comment]
I like what I see.

Just few fixes, we refactor deeper on V11.0

File: web_widget_json_graph/static/src/js/web_widget_json_graph.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+//
+//   This program is distributed in the hope that it will be useful,
+//   but WITHOUT ANY WARRANTY; without even the implied warranty of
+//   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//   GNU Affero General Public License for more details.
+//
+//   You should have received a copy of the GNU Affero General Public License
+//   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+//############################################################################
```
Comment (nhomar) [inline web_widget_json_graph/static/src/js/web_widget_json_graph.js]: User shorter licence please.

**Review by josemoralesp — APPROVED** [review]

**Review by nhomar — CHANGES_REQUESTED** [review]
You have lints problems yet related to your module, change them or ignore them explicitally please.

in the architecture I am agree on this approach.

**Review by nhomar — APPROVED** [review]

**Pull request merged**